src/devices/Llc200d3sh/README.md

```diff
@@ -0,0 +1,11 @@
+# Optomax digital liquid level switch
+
+Optomax digital liquid level switch is a device that can detect the presence of liquid/water. GPIO can be used to communicate with the device.
```

Comment:
I got other water switch sensors which I guess work identical - I discourage using specific device name here except for the case where you'd preconfigure the class, so i.e. you could have base class like `WaterLevelSensor` and derived class `Llc200d3sh` - I think it should still live somewhere in this folder considering how simple the implementation is

Reply:
for the sample you can show it on any sensor so basically the diagram can stay as is but if you go with two classes then please show how to configure the class yourself and how to use the specialized version

Reply:
Can you please supply the names and data sheets of some of the other liquid switch sensors you know about so I can have a look at their details.

Reply:
@DazFahy - I haven't got any spec for them, they were super cheap since they were basically breaking the circuit when water wasn't there (or opposite) so I only connected them between `High` and input and pulled down the input so when circuit was broken it was producing `Low`.

I think it was this one: https://www.aliexpress.com/item/4000265498553.html?spm=a2g0o.productlist.0.0.7d8c22abhyyPaK&algo_pvid=414159f0-5dd1-4c99-ac0a-245d4683c933&algo_expid=414159f0-5dd1-4c99-ac0a-245d4683c933-0&btsid=0be3769015911337997453345e8e02&ws_ab_test=searchweb0_0,searchweb201602_,searchweb201603_ or equivalent